docs/docs/guides/migration_from_other_libs/_category_.yml

```diff
@@ -1,4 +1,5 @@
-label: 'Migration from Other Libraries'
+label: '🔄 Migration from other Libraries'
```

Comment:
Its appearing on two lines , so should be renamed to 'Migration Guides' or some other text so its on one line 
<img width="271" alt="image" src="https://github.com/web3/web3.js/assets/86780488/e0c1799e-ea7a-4513-9573-1d78e919d1cd">